inst/plumber/v1/endpoints.R

```diff
@@ -9,23 +9,23 @@ library(pipapi)
 #* Ensure that version parameter is correct
 #* @filter validate_version
 function(req, res) {
-  # tictoc::tic("filters")
-  # browser()
-  if (!is.null(req$argsQuery$version) & !grepl("swagger", req$PATH_INFO)) {
-    if (!is.null(req$argsQuery$version)) {
-      if (!req$argsQuery$version %in% lkups$versions) {
+
+  #If no arguments are passed, use the latest version
+  if (is.null(req$argsQuery$release_version) && is.null(req$argsQuery$ppp_version) &&
+     is.null(req$argsQuery$version) && is.null(req$argsQuery$identity)) {
+      version <- lkups$latest_release
+  } else {
+    version <- pipapi::return_correct_version(req$argsQuery$version, req$argsQuery$release_version, req$argsQuery$ppp_version, req$argsQuery$identity, lkups$versions)
```

Comment:
I exported the function, otherwise it is not being found when the API is running